apps/mull-ui/src/app/pages/home/my-events/my-events.tsx

```diff
@@ -16,7 +15,12 @@ export const MyEventsPage = ({ history }) => {
       ISerializedEvent
     >[];
     var eventCards = events.map((event, index) => (
-      <EventCard key={index} event={event} onClick={() => history.push(`/events/${event.id}`)} />
+      <EventCard
+        isJoined={true}
```

Comment:
Since this tab is where all the events that the user has created, I think it should have its own condition to display. We can keep this here for now and update this later in another ticket